proto/api.go

```diff
@@ -188,18 +188,18 @@ func (rh *ResponseHeader) GoError() error {
 	// match the methods of the specific error type.
 	if rh.Error.Retryable {
 		if r, ok := err.(util.Retryable); !ok || !r.CanRetry() {
-			log.Fatalf("inconsistent error proto; expected %T to be retryable", err)
+			panic(fmt.Sprintf("inconsistent error proto; expected %T to be retryable", err))
```

Comment:
Import cycles. Now that the log package is getting more complicated, nothing that `log` depends on can actually use logging. This is not ideal and we should consider moving the logging protos into their own package (as we've done for the low-level rpc protos).